### ISSUE TITLE
Add missing flags

### ISSUE THREAD

**Issue by lwcorp** [Blazingstix/QFGImporter#9]
QFG2 is the only one in which you revealed all flags.

Since I see you started progress again, does it mean there's hope you'll uncover those mysterious flags...? I know you did it in the source code years ago, but you never made a new release (nor did you update the README about QFG3/4).
![image](https://user-images.githubusercontent.com/1773306/183242070-f7f065f7-d8eb-4a66-b035-dc78bdd64559.png)

QFG1:
Only 1 flag left to reveal:
![image](https://user-images.githubusercontent.com/1773306/183242036-70d60826-8ede-4fd2-a05b-8e6a23a0364f.png)

QFG2:
You already revealed all flags, how cool is that?
![image](https://user-images.githubusercontent.com/1773306/183242001-15749ab2-1b7e-41c9-8531-28ed2dbdcd48.png)

QFG3:
Only 1 item is revealed:
![image](https://user-images.githubusercontent.com/1773306/183241960-49bd0efa-d423-4487-bb1b-9ba6ab3914b4.png)

QFG4:
None of the flags are revealed:
![image](https://user-images.githubusercontent.com/1773306/183241986-4aed78fa-bbf8-41a4-a8e9-27a1911d2e9b.png)

**Comment by lwcorp** [comment]
Here's the entire solution:

**QFG1:**
* Flag **7** is **Got normal score (> 255)** (out of 500).
* Can you fix **Hero of Spielburg** to just **Got hand mirror**? That's all it is. In theory it could mean you still would have had the mirror in future games.

Code references:
1. https://github.com/EricOakford/SCI-Decompilation-Archive/blob/master/qfg1vga/src/CharSave.sc#L148
1. https://github.com/sluicebox/sci-scripts/blob/main/qfg1vga-dos-2.000-patched/src/CharSave.sc#L154

**QFG2:**
Already perfect, nothing needs to be done.

Code reference: https://github.com/sluicebox/sci-scripts/blob/main/qfg2-dos-1.105/src/CharSave.sc#L191

**QFG3:**
* Flag **1** is **Dispel potions**.
* Flag **2** is **Tinderbox**.
* Flag **4** is **Fine Dagger**.
* Flag **5** is **Waterskin**.
* Flag **6** is **Blackbird**.
* Flag **7** is **Defeated Baba Yaga in QFG1**. This can normally only be enabled if imported directly from QFG1, as QFG2 didn't export this info...(actually, QFG3 is the only sequel that exports this QFG1 info).
* Flag **8** is **Rope**.
* Flag **9** is **Thief's Toolkit**.
* Flag **10** is **Sword**.
* Flag **11** is **Magic Shield** (but for those who cheated, the game doesn't export it unless it's a Fighter - I guess the game forgot Paladins are supposed to have it too).
* According to the code, it seems you should rename flags **12-16** to just **(unused)** or maybe just delete them?

Code references:
1. https://github.com/EricOakford/SCI-Decompilation-Archive/blob/master/qfg3/src/CharSave.sc#L189
1. https://github.com/sluicebox/sci-scripts/blob/main/qfg3-dos-1.1/src/CharSave.sc#L156

**QFG4:**
* Flag **5**, which your program's save files set by default, is **Had Black Bird in QFG3**.
* Flag **6** is **Got Black Bird**.
* According to the code, it seems you should rename flags **1-4** and **8-16** to just **(unused)** and maybe even delete flags 1-4?

Code references:
1. https://github.com/EricOakford/SCI-Decompilation-Archive/blob/master/qfg4cd/src/CharSave.sc#L174
1. https://github.com/sluicebox/sci-scripts/blob/main/qfg4-cd-dos-1.0/src/CharSave.sc#L152

**Comment by Blazingstix** [comment]
Thanks for the detailed legwork, Iwcorp.
I had forgotten that I had already implemented almost all of your Flag renaming suggestions into the source code, but I had not published them before.

I took the opportunity now to dig into the final remaining unknown items, and I have fully identified everything.  I used a combination of ErikOakford's decompilation archive that you referenced, as well as decompilations of my own.

And I have now finally published the latest version. V2.0.
